community/libbsd/depends

```diff
@@ -0,0 +1 @@
+autoconf
```

Comment:
This whole file can be removed.

Reply:
Can it? I thought they always needed a depends file.

Reply:
The `depends` file is optional and can be omitted entirely if unneeded.